mdpow/templates/em_opls.mdp

```diff
@@ -1,4 +1,4 @@
-; Gromacs mdp file
+; GROMACS mdp file
 ; Template for energy minimization with OPLS-AA (part of MDPOW)
```

Comment:
Same comment as above: OPLS-AA .mdp files are also used for AMBER/GAFF, in which case the description will not be correct. I would put something like "OPLS-AA or AMBER". Maybe also change the name of the file into "em_opls_amber.mdp", which is more clear.

Reply:
made amber version